content/en/docs/installation/supported-releases.md

```diff
@@ -0,0 +1,269 @@
+nightly can be seen in [`previous-periodics.yaml`][] and
+[`next-periodics.yaml`][], and the results are visible at
+[`testgrid-previous`][] and [`testgrid-next`][]).
+
+[next-periodics.yaml]: https://github.com/jetstack/testing/blob/master/config/jobs/cert-manager/release-next/cert-manager-release-next-periodics.yaml
+[previous-periodics.yaml]: https://github.com/jetstack/testing/blob/master/config/jobs/cert-manager/release-previous/cert-manager-release-previous-periodics.yaml
+[testgrid-previous]: https://testgrid.k8s.io/jetstack-cert-manager-previous
+[testgrid-next]: https://testgrid.k8s.io/jetstack-cert-manager-next
+
+#### Security issues {#security-issues}
+
+**Security issues** are fixed as soon as possible. They get back-ported to
+the last two releases, and a new patch release is immediately created for
+them:
+
+```text
+  v1.1.0                                           v1.1.1
+----+-------------------------------------------------+-----> release-1.1
+     \                                       backport ^
+      \                                      commit  /
+       \                                            /
+        \                                          /
+         \                                        /
+          \         v1.2.0               v1.2.1  /
+           ------------+---------------------+--/-----------> release-1.2
+                        \           backport ^ /
+                         \          commit  / /
+                          \                / /
+                           \              / /
+                            \            / /
+                             \          / /
+                              ------------------------------> master
+                                     ^   ^
+                            security |   | two new patch
+                               patch |   | releases
+                              merged       created immediately
+```
+
+<!-- Diagram source: https://textik.com/#7c4096204b3c0ad3 -->
+
+#### Critical bugs {#critical-bugs}
+
+**Critical bugs** include both regression bugs as well as upgrade bugs.
+
+Regressions are functionalities that did work in the previous release but
+no longer works. [#3393][] and [#2857][] are two examples of regressions.
+
+Upgrade bugs are bugs (often Helm-related) preventing users from upgrading
+to earlier final releases of cert-manager. [#3882][] and [#3644][] are
+example of such upgrade bugs.
+
+The fixes are back-ported; we then wait until the next final release for
+trigger a patch release. The patch release will be made for all supported
+release. Note that since we do only support the two last releases, patch
+releases are only made to the last release.
+
+In the below example, the release branches `release-1.2` and `release-1.3`
+are the two supported releases at the time of the release of 1.3.0, which
+means that the critical bug fix 1 and 2 will only be back-ported to
```

Comment:
suggestion: wording/grammar

```suggestion
means that critical bug fixes 1 and 2 will only be back-ported to
```

Reply:
Fixed in 71647eee926968ebc20283a2bb108222ab9e7829